app/templates/components/validated-input.hbs

```diff
@@ -0,0 +1,24 @@
+{{!-- Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies --}}
```

Comment:
Yup, just looking out for the merge conflicts. :) 

Since this is a dummy app helper, there's probably no other way to pull this in, and duplicating code from the addon is fine in that case. If you find the component to be that useful, someday you might consider sending a PR to the authors making the component available by default.